runners/phpunit-core/script.sh

```diff
@@ -11,4 +11,4 @@ ARGS=( $DRUPAL_TI_PHPUNIT_ARGS )
 
 # Run core tests
 cd core
```

Comment:
We can, but then we need to add a "-c core" to the phpunit invocation. Thought that might interfere with "${ARGS[@]}", though it's not very likely.